packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -33,7 +33,7 @@ function PostLink( {
 } ) {
 	const { prefix, suffix } = permalinkParts;
 	let prefixElement, postNameElement, suffixElement;
-	const currentSlug = postSlug || cleanForSlug( postTitle ) || postID;
+	const currentSlug = decodeURIComponent( postSlug ) || cleanForSlug( postTitle ) || postID;
```

Comment:
@iseulde in other places we use `safeDecodeURI` from `@wordpress/url` package, should we introduce something similar for `decodeURIComponent`?

Reply:
I think that's probably a good idea. If something else modifies `post_name` directly and there was an issue with the encoded string, it would lead to a full editor crash. Better in that case just to return the encoded string. 

@Naerriel can you add a new `safeDecodeURIComponent` function to use for this with `safeDecodeURI` as a model?

Reply:
Naturally. Thank you for feedback.